File: 02-frontend/react-library/src/models/AddBookRequest.ts
export default class AddBookRequest{

    title:string;
    author:string;
    description:string;
    category:string;
    copies:number;
    copiesAvailable:number;
    image:any;

    constructor(title:string,author:string,description:string,category:string,
        copies:number,image:any){
            this.title = title;
            this.author = author;
            this.description = description;
            this.category = category;
            this.copies = copies;
            this.copiesAvailable = copies;
            this.image = image;
        }
}

File: 03-backend/spring-boot-library/src/main/java/com/luv2code/springbootlibrary/requestmodels/AddBookRequest.java
package com.luv2code.springbootlibrary.requestmodels;

import lombok.Data;

@Data
public class AddBookRequest {
    
    private String title;

    private String author;

    private String description;

    private int copies;

    private int copiesAvailable;

    private String category;

    private String image;
}
